docs/source/index.rst

```diff
@@ -16,7 +16,7 @@ Contents:
 .. toctree::
    :maxdepth: 2
 
-   VUDRM-Token-Integration.md
+   VUDRM-token.md
```

Comment:
Is this intended to be lowercase "token" and not capitalised like the rest?

Reply:
lower case. It is the filename.